src/desktop/lan.desktop

```diff
@@ -6,12 +6,12 @@ X-KDE-ModuleType=Library
 X-KDE-HasReadOnlyMode=true
 X-SuSE-YaST-Call=lan
 
-X-SuSE-YaST-Group=Network
+X-SuSE-YaST-Group=System
 X-SuSE-YaST-Argument=
 X-SuSE-YaST-RootOnly=true
 X-SuSE-YaST-AutoInst=all
 X-SuSE-YaST-Geometry=
-X-SuSE-YaST-SortKey=
+X-SuSE-YaST-SortKey=!first
```

Comment:
I think it would make sense for it to be the first only if it got to the Network Services group, as their "prerequisite". But in the System group, it should get no special treatment.